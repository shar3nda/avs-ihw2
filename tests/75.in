a$mxB[pb61.0hQu
2{GRDts<fdf,}gq4P6$zb9Yz7tL;^yEnt8;D$P2J}-EJs'e)_W5toU~//>I3q