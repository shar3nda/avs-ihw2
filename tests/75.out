Lowercase: 26, uppercase: 17